### ISSUE TITLE
Have `lycheeVersion: latest` to use the current lychee release

### ISSUE THREAD

**Issue by tooomm** [lycheeverse/lychee-action#256]
As discussed earlier in a related ticket to https://github.com/lycheeverse/lychee/pull/1212:
https://github.com/lycheeverse/lychee-action/issues/203

See comments following https://github.com/lycheeverse/lychee-action/issues/203#issuecomment-1784083059, including a proposed logic by @kemingy.

**Comment by mre** [comment]
Wouldn't this update the lychee version even for users who've pinned the lychee-action? This could potentially introduce breaking changes unexpectedly.

I'm not entirely sure how GitHub Actions handle this interaction between action versions and tool versions.

**Comment by mre** [comment]
@tooomm, any thoughts?

**Comment by tooomm** [comment]
>Wouldn't this update the lychee version even for users who've pinned the lychee-action? This could potentially introduce breaking changes unexpectedly.

Yeah, it's probably safer to control the tool version via this action repo and have an automation in place to help with the maintenance keeping it en par with the source.
I think #284 is a good approach and we can close this ticket even. 👍

**Comment by mre** [comment]
Yeah, @Arteiii did a great job there. Glad that works for you. Closing. 😊
